tools/arriba-drawing/2.4.0/arriba-drawing__2.4.0.cwl

```diff
@@ -26,7 +26,7 @@ hints:
   ResourceRequirement:
     ilmn-tes:resources/tier: standard
     ilmn-tes:resources/type: standard
-    ilmn-tes:resources/size: xlarge
+    ilmn-tes:resources/size: xxlarge
     coresMin: 4
     ramMin: 14000
```

Comment:
Should now be

```yaml
coresMin: 8
ramMin: 28000
```

As per https://illumina.gitbook.io/ica-v1/analysis/a-taskexecution

Reply:
updated in https://github.com/umccr/cwl-ica/commit/7c0f499d20492d39214be99c2d5055b36d9e365b